packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -122,7 +122,7 @@ describe("eventstream event data", () => {
       const exampleDataJson = JSON.parse(exampleDataStr.slice(5).trim()) as unknown;
 
       const testEvent = eventTestData[topic as keyof typeof eventTestData];
-      if (testEvent == null) {
+      if (testEvent === null) {
```

Comment:
the `topic as keyof typeof eventTestData` in `eventTestData[topic as keyof typeof eventTestData]` ensures that `testEvent` will have a non-null/non-undefined value...if not, this fails at compile time. Hence the `testEvent === null)` check might even be considered redundant.